trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/index.test.ts

```diff
@@ -149,8 +149,8 @@ describe('controllers/utilisation-reports/utilisation-report-reconciliation-for-
         enablePaymentsReceivedSorting: true,
         reportId: '1',
         feeRecordPaymentGroups: feeRecordPaymentGroupViewModel,
-        premiumPaymentFormError: undefined,
-        facilityIdQueryError: undefined,
+        tableDataError: undefined,
+        filterError: undefined,
```

Comment:
Looks like we shoudn't need these anymore since `UtilisationReportReconciliationForReportViewModel` has these as optional?